src/main/java/edu/rosehulman/covidtracer/model/Role.java

```diff
@@ -15,21 +15,56 @@
 	private Long ID;
 
 	@Column(name = "Role", nullable = false, unique = true)
-	private String role;
+	private ROLE role;
 
-	public Role() {
-	}
+	public Role() {}
 
 	public Role(String role) {
-		this.role = role;
+		this.role = ROLE.roleFromString(role);
 	}
 
 	public Long getID() {
 		return ID;
 	}
 
 	public String getRole() {
-		return role;
+		return role.toString();
 	}
 
 }
+
+enum ROLE {
+	PERSON, HEALTH_SERVICES, CONTACT_TRACER, HEAD_CONTACT_TRACER, STUDENT_AFFAIRS;
```

Comment:
Can We add in a system admin or an admin role that has access to eerythang?

Reply:
Yes thanks for the reminder! How do the others look?

Reply:
Others look great! :)